code/tables/0_summary_table.do

```diff
@@ -35,19 +38,52 @@ file open f using "$repository/code/tables/output/summary_table.tex", write repl
 
 // write the beginning of the table
 file write f "\begin{table}[htbp]" _n ///
-"\caption{Title Me Here}" _n ///
+"\caption{Summary Statistics By Host Race: Listing Characteristics}" _n ///
 "\begin{center}%" _n ///
-"\small\begin{tabular}{l c c}" _n
+"\small\begin{tabular}{l c c c c c c}" _n
 
 // write column headers
 file write f " &  & All" _n
 file write f "\\" _n ///
 "\hline\hline\noalign{\smallskip} " _n
 
 // write in stats
-// hi pick up here
-
-
+local ncat price number_of_reviews accommodates bedrooms bathrooms beds amenities cleaning_fee extra_people minimum_nights first_review
+local cat property_type room_type instant_bookable cancellation_policy
+
+	foreach i in `ncat'{ //loops over noncategorical variables
+			levelsof race
+			foreach f in `r(levels)' {
+				sum `i' if race == "`f'"
+				loc `f'_mean_`i' = `r(mean)'
+			}
+			//write means
+			//write label
+			file write f " `: var label `i'' & " %4.3f (`NO_mean_`i'') " & " %4.3f (`YES_mean_`i'') " "
```

Comment:
`file write f " `: var label `i'' & " %4.3f (`White_mean_`i'') " & " %4.3f (`Black_mean_`i'') " "`

Reply:
instead of NO_mean_i and YES_mean_i, you need the levelsof race. Because check out where that local is created in line #58 - it starts with `f', which is the level of race. so you're going to be 4? 5? locals, all starting with black_mean_nameofvariablehere, white_mean_nameofvariablehere, hispanic_mean_nameofvariablehere